playground/static/main.js

```diff
@@ -158,32 +158,56 @@ function toggleView() {
 }
 
 async function loadPackageInfo() {
-    const info = JSON.parse(await compilerAction({type: "package_info"}));
+    const info = JSON.parse(await compilerAction({ type: "package_info" }));
 
-    const type_str = (type) => {
+    const type_annotation = (type) => {
+        let type_name = type;
+        let color = "";
         if (Array.isArray(type)) {
-            return type.join("/");
+            type_name = type.join("/");
+            color = "#90A959";
+        } else if (type == "String") {
+            color = "#2A6041";
+        } else if (type == "Unsigned integer") {
+            color = "#3581B8";
+        } else if (type == "Integer") {
+            color = "#8B85C1";
+        } else if (type == "Float") {
+            color = "#C3423F";
+        }
+
+        return `<span style="background-color: ${color};" class="type">${type_name}</span>`
+    }
+
+    const escape_default = (type, default_value) => {
+        if (Array.isArray(type) || type == "String") {
```

Comment:
Suggestion: Disable type coercion
```suggestion
        if (Array.isArray(type) || type === "String") {
```